frontend_server/BUILD.gn

```diff
@@ -71,4 +74,118 @@ if (is_fuchsia_host || is_fuchsia) {
 
     inputs = frontend_server_files
   }
+
+  # For flutter/flutter#36738 we make the source files availible so that
+  # we can generate a local frontend_server snapshot in the tools cache.
+
+  # Creates a packageable vm.
+  prebuilt_dart_action("flutter_package_vm") {
```

Comment:
I understand the desire to use Dart in `publish.dart` but this template invocation now uses the Python to list the files, Dart to publish the same, and compiles the host VM (~2000 TUs) to boot. We do have a prebuilt VM you can use for complicated Dart scripts but I'd rather you just replace these steps with a single script action. That will be faster in cases where this copy step is the only step necessary (since you wont have to build the VM) and can be parallelized by the build system when other targets that need the VM are also being built.

Reply:
Understood, I'll update the script to use python.

Reply:
FYI `prebuilt_dart_action()` should be using the prebuilt Dart VM. If it isn't then that is a bug.

Reply:
You're right. I was thinking of `dart_action`. I suppose I am now significantly less concerned about this but I still think it'd be cleaner and less verbose to just have a single `action` target that achieves this instead of a target definition that mixes both python and dart with a custom template.

Reply:
What would be the resolution here - to write this as a single target in python? dart?

Reply:
@chinmaygarde Would a single python script wrapped in a GN template scoped to this file address your concerns?

Reply:
Yes. A single python script would be great. Thanks.

Reply:
Updated to use a single/action python script.

Reply:
Updated to use a single python script